ddtrace/internal/agent.py

```diff
@@ -74,12 +75,13 @@ def get_connection(url, timeout=DEFAULT_TIMEOUT):
     """Return an HTTP connection to the given URL."""
     parsed = verify_url(url)
     hostname = parsed.hostname or ""
+    path = parsed.path or "/"
 
     if parsed.scheme == "https":
-        return compat.httplib.HTTPSConnection(hostname, parsed.port, timeout=timeout)
+        return HTTPSConnection.with_base_path(hostname, parsed.port, base_path=path, timeout=timeout)
     elif parsed.scheme == "http":
-        return compat.httplib.HTTPConnection(hostname, parsed.port, timeout=timeout)
+        return HTTPConnection.with_base_path(hostname, parsed.port, base_path=path, timeout=timeout)
     elif parsed.scheme == "unix":
-        return UDSHTTPConnection(parsed.path, hostname, parsed.port, timeout=timeout)
+        return UDSHTTPConnection(path, hostname, parsed.port, timeout=timeout)
```

Comment:
Not sure how big a problem this is but we don't support UDS with a base path.

Reply:
As mentioned in the PR description it is not possible to use the agent URL path here because the path is used for the unix socket path on the filesystem.